### PR TITLE
CODE RUB: Replace default with Guid.Empty

### PR DESCRIPTION
**Pull request by KFitz114**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianAttachments/GuardianAttachmentServiceTests.Validations.Remove.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public async Task ShouldThrowValidatonExceptionOnRemoveWhenGuardianIdIsInvalidAn
         {
             // given
             Guid randomAttachmentId = Guid.NewGuid();
-            Guid randomGuardianId = default;
+            Guid randomGuardianId = Guid.Empty;
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianAttachments/GuardianAttachmentServiceTests.Validations.Remove.cs:22]: @KFitz114 we should rename this to `invalidGuardianId`

Comment (KFitz114) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianAttachments/GuardianAttachmentServiceTests.Validations.Remove.cs]: @hassanhabib updated to change variable name from "random" to "invalid".

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianAttachments/GuardianAttachmentServiceTests.Validations.Remove.cs [hunk 2]
```diff
@@ -19,9 +19,9 @@ public async Task ShouldThrowValidatonExceptionOnRemoveWhenGuardianIdIsInvalidAn
         {
             // given
             Guid randomAttachmentId = Guid.NewGuid();
-            Guid randomGuardianId = default;
+            Guid invalidGuardianId = Guid.Empty;
             Guid inputAttachmentId = randomAttachmentId;
-            Guid inputGuardianId = randomGuardianId;
+            Guid inputGuardianId = invalidGuardianId;
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianAttachments/GuardianAttachmentServiceTests.Validations.Remove.cs:24]: We do not need this variable.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by ShriHumrudha** [comment]
Closing the PR as it is beyond the 3-day window.

**Pull request closed without merging**